File: pgm/ABC_Dart/ABC_405_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [int R, int X] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const int lower1 = 1600;
  const int lower2 = 1200;
  const int upper1 = 2999;
  const int upper2 = 2399;
  const String Yes = 'Yes';
  const String No = 'No';
  const List<int> lower = [lower1, lower2];
  const List<int> upper = [upper1, upper2];
  print(lower[X - 1] <= R && R <= upper[X - 1] ? Yes : No);
  exit(0);
}

File: pgm/ABC_R/ABC_405_A.r
con <- file(description = "stdin", open = "r")
arr <- readLines(con = con)
close(con)
Yes <- "Yes"
No  <- "No"
blank <- ""
space <- " "
yn <- "\n"
one <- 1L
two <- 2L
lower1 <- 1600L
lower2 <- 1200L
upper1 <- 2999L
upper2 <- 2399L
lower <- c(lower1, lower2)
upper <- c(upper1, upper2)
R <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][one] |>
  as.integer()
X <- strsplit(x = arr[one], split = space, fixed = TRUE)[[one]][two] |>
  as.integer()
if ((lower[X] <= R) && (R <= upper[X])) {
  Yes |>
    cat(yn, sep = blank)
} else {
  No |>
    cat(yn, sep = blank)
}
q("no")